module/programs/helper.js

```diff
@@ -438,4 +439,274 @@ module.exports = class ProgramsHelper {
     })
   }
 
+     /**
+   * 
+   * @method
+   * @name mapObservation
+   * @param {String} programId - Program Id.
+   * @param {Array} entities - entities.
+   * @returns {JSON} - Removed entities data.
+   */
+
+      static mapObservation(programId, questionSetId, copyReq) {
+        return new Promise(async (resolve, reject) => {
+          try {
+            let status;
+            console.log("programId", programId, copyReq, questionSetId)
+            let programData = await database.models.programs.findOne({ _id: programId }).lean();
+            if (!programData) {
+              return resolve({
+                message: messageConstants.apiResponses.PROGRAM_NOT_FOUND,
+                status: httpStatusCode.bad_request.status,
+              });
+            }
+            console.log(" database.model",  programData)
+            const copyQuestionSetUrl = kpServiceUrl + messageConstants.endpoints.COPY_QUESTION_SET + "/" + questionSetId;
+            const headers = {
```

Comment:
@aishwa8141 Please service folder inside generics for all external service calls - https://github.com/shikshalokam/ml-survey-service/tree/master/generics/services